torch/__init__.py

```diff
@@ -2270,7 +2270,7 @@ def compiled_with_cxx11_abi() -> builtins.bool:
     matrix_rank,
     solve,
 )
-from torch.utils.dlpack import from_dlpack, to_dlpack
+from torch.utils.dlpack import from_dlpack, to_dlpack, to_dlpack_unversioned
```

Comment:
While I understand this matches current code, I don't think we want to have this as a new top level API? Being part of torch.utils.dlpack is enough?